modules/mobile/app/helpers/mobile/appointments_cache_interface.rb

```diff
@@ -51,10 +53,23 @@ def fetch_from_external_service(user, start_date, end_date)
     # must break the cache if user is requesting dates beyond default range to ensure the integrity of the cache.
     # at this time, it's not possible for the user to fetch beyond this range because the interface doesn't allow it,
     # so the cache will effectively always be from beginning of last year until one year from today
-    def use_cache?(start_date, end_date, use_cache)
-      use_cache &&
-        start_date == latest_allowable_cache_start_date &&
-        end_date == earliest_allowable_cache_end_date
+    def fetch_cache?(start_date, end_date, fetch_cache)
```

Comment:
Renaming this because use_cache is a bit ambiguous, given that we still set the cache if we're not fetching from it.